rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -805,6 +809,235 @@ private void assertEventWasEmittedWithExpectedData(byte[] expectedData) {
                 .findFirst();
             assertTrue(data.isPresent());
         }
+
+        @Test
```

Comment:
I believe we have a specific class for registerBtcTransaction tests. `BridgeSupportRegisterBtcTransactionTest`. Should we move these tests there?

Reply:
it makes sense actually. But im not sure 🙃

Reply:
I agree with @marcos-iov